contracts/deploy/069_oeth_to_ogv_governance_p2.js

```diff
@@ -0,0 +1,78 @@
+const { deploymentWithGovernanceProposal } = require("../utils/deploy");
+const addresses = require("../utils/addresses");
```

Comment:
nitpick: addresses is not used in this script

Reply:
thanks removing